docs/APISpec.md

```diff
@@ -0,0 +1,185 @@
+# APISpec
+
+## Usage
```

Comment:
We could again link to upstream docs and only mention differences
<https://github.com/anuraghazra/github-readme-stats/tree/678fa1141676fc8fb75e2a322e4297323900e330#common-options>
<https://github.com/anuraghazra/github-readme-stats/tree/678fa1141676fc8fb75e2a322e4297323900e330#wakatime-card-exclusive-options>